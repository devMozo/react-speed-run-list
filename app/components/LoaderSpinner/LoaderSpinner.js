import './LoaderSpinner.scss';

export default () => <p className="LoaderSpinner"> Loading... </p>;
