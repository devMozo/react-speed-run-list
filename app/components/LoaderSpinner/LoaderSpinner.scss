@import '../../fonts/index.scss';

.LoaderSpinner {
  width: 100%;
  padding: 20px 0;
  text-align: center;
  color: #ccc;
  font-family: $fontMedium;
}
