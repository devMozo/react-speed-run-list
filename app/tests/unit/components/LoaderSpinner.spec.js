import React from 'react';
import { shallow } from 'enzyme';
import LoaderSpinner from '../../../components/LoaderSpinner/LoaderSpinner';

describe('<LoaderSpinner />', () => {
  it('should render correctly', () => {
    const component = shallow(<LoaderSpinner />);
    expect(component).toMatchSnapshot();
  });
});
