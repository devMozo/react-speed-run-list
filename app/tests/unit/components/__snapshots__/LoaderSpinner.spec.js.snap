// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoaderSpinner /> should render correctly 1`] = `
<p
  className="LoaderSpinner"
>
   Loading... 
</p>
`;
